packages/mdc-textfield/foundation.js

```diff
@@ -57,6 +57,7 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
     this.inputBlurHandler_ = () => this.deactivateFocus_();
     this.inputInputHandler_ = () => this.autoCompleteFocus_();
     this.inputKeydownHandler_ = () => this.receivedUserInput_ = true;
+    this.useNativeValidityChecking_ = true;
```

Comment:
IMO "Custom" and defaulting to `false` might be easier to understand anyway. i.e. the default behavior is no custom validity (use native HTML validity checking).  The non-default is adding something moreso than removing something.